test/os2lana_cfg.py

```diff
@@ -291,6 +299,7 @@
     *process.evtcleaner
     *process.cleanedEvents
     *process.ana
+#    *process.massReco
     *process.finalEvents
     )
```

Comment:
@tmitchel when you run the mass reco, you always run on the skims, right?

in that case, do you need to run all the other modules in your path every time?